.travis.yml

```diff
@@ -19,29 +19,28 @@ d:
   # this way the overall test time gets cut down (GDC/LDC are a lot
   # slower tham DMD, so they should be started early), while still
   # catching most DMD version related build failures early
-  - dmd-2.074.0-rc1
+  - dmd-2.074.0
   - dmd-2.070.2
-  - ldc-1.1.1
```

Comment:
The [build matrix](https://travis-ci.org/rejectedsoftware/vibe.d/builds/237027475) is already extremely long. Do you have any specific reason to keep ldc-1.1.1?